source/val/validate_debug.cpp

```diff
@@ -51,9 +53,33 @@ spv_result_t ValidateLine(ValidationState_t& _, const Instruction* inst) {
   return SPV_SUCCESS;
 }
 
+bool IsDebugInstruction(const Instruction* inst) {
```

Comment:
This looks useful in general. I would suggest moving it into opcode.cpp/h and naming spvOpcodeIsDebugInst

Reply:
Done, dropped the Inst suffix to match the style of other functions in the header.